features/admin.authentication.v1/hooks/use-sign-in.ts

```diff
@@ -125,38 +125,74 @@ const useSignIn = (): UseSignInInterface => {
      * @param orgType - Type of the organization. Ex: sub organization, etc.
      */
     const setCustomServerHost = (orgType: string): void => {
-        // In case of failure customServerHost is set to the serverHost.
-        let customServerHost: string = Config?.getDeploymentConfig()?.serverHost;
+        const disabledFeatures: string[] = window["AppUtils"]?.getConfig()?.ui?.features?.branding?.disabledFeatures;
```

Comment:
```suggestion
        const disabledBrandingFeatures: string[] = window["AppUtils"]?.getConfig()?.ui?.features?.branding?.disabledFeatures;
```

If possible, let use the useFeatureConfig() hook instead of accessing the window["AppUtils"] object.

Reply:
updated with [b30512c](https://github.com/wso2/identity-apps/pull/6367/commits/b30512c6a8581a070638c0b49724a8304e7e1b49)